package.json

```diff
@@ -11,8 +11,10 @@
     "William Earnhardt (https://wearnhardt.com)"
   ],
   "dependencies": {
-    "@newfold-labs/wp-module-runtime": "^1.0.0",
+    "@heroicons/react": "^2.0.18",
```

Comment:
@AleTorrisi I think we can remove this `@heroicons/react`, I don't see it being used anywhere.